test/integration/testImage/src/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/heptio/sonobuoy/test/integration/testImage/src
+
+go 1.12
```

Comment:
Can this use go 1.13? That's what is being used everywhere else.

Reply:
👍 I dont know how much it matters; it had to do with the fact that I ran `go mod init` locally with go 1.12 but after updating go and such to get this to be the go 1.13, the go.sum/go.mod doesnt change in any way (just FYI, the requested change was made)